src/main/scala/ci/release/early/Utils.scala

```diff
@@ -12,31 +12,31 @@ case class VersionAndTag(version: String, tag: String)
 
 object Utils {
 
-  def determineAndTagTargetVersion: VersionAndTag = {
+  def determineAndTagTargetVersion(log: String => Any): VersionAndTag = {
```

Comment:
shouldn't that be `String => Unit`?

Reply:
I was just thinking of loggers with signatures like `def info(msg: String): SomeAsyncLogFeedback`. Ping me if you really think this should change, for this PR I'll leave it as is